app/src/main/res/layout/fragment_sessions.xml

```diff
@@ -85,6 +86,21 @@
                 android:textColor="@color/white"
                 />
 
+        <FrameLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:background="@color/white"
```

Comment:
I found an issue to update visibility in doOnSuccess/doOnError.
doOnSuccess and doOnError run on not-UI thread therefore notifyPropertyChanged causes CalledFromWrongThreadException.
Of course I can notify with Handler... what should I do?